Userland/Libraries/LibGUI/Toolbar.cpp

```diff
@@ -142,4 +142,16 @@ Optional<UISize> Toolbar::calculated_preferred_size() const
     VERIFY_NOT_REACHED();
 }
 
+Optional<UISize> Toolbar::calculated_min_size() const
+{
+    if (m_collapsible) {
```

Comment:
The `collapsible` concept of a `Toolbar` is not yet defined at this point. You introduce it 2 commits later.